2015/02/12 11:07:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin/login".' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('admin/login')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/venturepact/index.php?r=admin/login
---
2015/02/12 11:08:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin/user/login".' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('admin/user/logi...')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/venturepact/index.php?r=admin/user/login
---
2015/02/12 11:12:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "clients".' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('clients')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/venturepact/index.php?r=clients
---
2015/02/12 11:13:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "users/login".' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('users/login')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/venturepact/index.php?r=users/login
---
2015/02/12 11:28:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "users/login".' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('users/login')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/venturepact/index.php?r=users/login
---
2015/02/13 05:16:38 [error] [php] include(ForgotpasswordForm.php): failed to open stream: No such file or directory (C:\xampp\htdocs\Yii\Yii1.1\test\framework\YiiBase.php:432)
Stack trace:
#0 C:\xampp\htdocs\venturepact\protected\controllers\SiteController.php(118): spl_autoload_call()
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#2 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(308): CInlineAction->runWithParams()
#3 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(133): SiteController->runAction()
#4 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(1145): CAccessControlFilter->filter()
#6 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CInlineFilter.php(58): SiteController->filterAccessControl()
#7 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#8 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(291): CFilterChain->run()
#9 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(282): SiteController->run()
#11 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#13 C:\xampp\htdocs\venturepact\index.php(13): CWebApplication->run()
REQUEST_URI=/venturepact/index.php?r=site/login
in C:\xampp\htdocs\venturepact\protected\controllers\SiteController.php (118)
in C:\xampp\htdocs\venturepact\index.php (13)
2015/02/13 05:19:11 [error] [exception.CException] exception 'CException' with message 'Property "Users.roles" is not defined.' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\db\ar\CActiveRecord.php(145): CComponent->__get('roles')
#1 C:\xampp\htdocs\venturepact\protected\components\UserIdentity.php(67): CActiveRecord->__get('roles')
#2 C:\xampp\htdocs\venturepact\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\xampp\htdocs\Yii\Yii1.1\test\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\htdocs\Yii\Yii1.1\test\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\xampp\htdocs\venturepact\protected\controllers\SiteController.php(123): CModel->validate()
#7 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilter.php(40): CFilterChain->run()
#11 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#13 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(291): CFilterChain->run()
#15 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(282): CController->run('login')
#17 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#18 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#19 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/venturepact/index.php?r=site/login
HTTP_REFERER=http://localhost/venturepact/index.php?r=site/login
---
2015/02/13 05:21:53 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\venturepact\protected\components\UserIdentity.php:67)
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CModel.php(159): CInlineValidator->validate()
#2 C:\xampp\htdocs\venturepact\protected\controllers\SiteController.php(123): LoginForm->validate()
#3 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#4 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(133): SiteController->runAction()
#6 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CInlineFilter.php(58): SiteController->filterAccessControl()
#9 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(265): SiteController->runActionWithFilters()
#12 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(282): SiteController->run()
#13 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\venturepact\index.php(13): CWebApplication->run()
REQUEST_URI=/venturepact/index.php?r=site/login
in C:\xampp\htdocs\venturepact\protected\components\UserIdentity.php (67)
in C:\xampp\htdocs\venturepact\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\venturepact\protected\controllers\SiteController.php (123)
2015/02/13 05:22:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "3".' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('3')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/venturepact/index.php?r=3
HTTP_REFERER=http://localhost/venturepact/index.php?r=site/login
---
2015/02/13 05:22:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "3".' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('3')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/venturepact/index.php?r=3
---
2015/02/13 05:27:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "page".' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(270): CController->missingAction('page')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(282): CController->run('page')
#2 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#3 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/venturepact/index.php?r=site/page&view=tables
HTTP_REFERER=http://localhost/venturepact/index.php?r=site/index
---
2015/02/13 05:27:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "page".' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(270): CController->missingAction('page')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(282): CController->run('page')
#2 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#3 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/venturepact/index.php?r=site/page&view=tables
HTTP_REFERER=http://localhost/venturepact/index.php?r=site/index
---
2015/02/13 08:55:55 [error] [php] Missing argument 1 for CHttpRequest::getPost(), called in C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CComponent.php on line 111 and defined (C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CHttpRequest.php:199)
Stack trace:
#0 C:\xampp\htdocs\venturepact\protected\controllers\SiteController.php(119): CHttpRequest->__call()
#1 C:\xampp\htdocs\venturepact\protected\controllers\SiteController.php(119): CHttpRequest->post()
#2 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#3 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(133): SiteController->runAction()
#5 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(1145): CAccessControlFilter->filter()
#7 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CInlineFilter.php(58): SiteController->filterAccessControl()
#8 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#9 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(291): CFilterChain->run()
#10 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(265): SiteController->runActionWithFilters()
#11 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(282): SiteController->run()
#12 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController()
#13 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#14 C:\xampp\htdocs\venturepact\index.php(13): CWebApplication->run()
REQUEST_URI=/venturepact/index.php?r=site/login
in C:\xampp\htdocs\venturepact\protected\controllers\SiteController.php (119)
in C:\xampp\htdocs\venturepact\protected\controllers\SiteController.php (119)
in C:\xampp\htdocs\venturepact\index.php (13)
2015/02/14 11:36:02 [error] [exception.CException] exception 'CException' with message 'Property "CActiveForm.role" is not defined.' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWidgetFactory.php(161): CComponent->__set('role', 'form')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(146): CWidgetFactory->createWidget(Object(SiteController), 'CActiveForm', Array)
#2 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(197): CBaseController->createWidget('CActiveForm', Array)
#3 C:\xampp\htdocs\venturepact\themes\admin\views\site\login.php(18): CBaseController->beginWidget('CActiveForm', Array)
#4 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#5 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(782): CController->renderPartial('login', Array, true)
#8 C:\xampp\htdocs\venturepact\protected\controllers\SiteController.php(152): CController->render('login', Array)
#9 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#10 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(282): CController->run('login')
#19 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#20 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#21 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/venturepact/index.php?r=site/login
---
2015/02/14 11:36:48 [error] [exception.CException] exception 'CException' with message 'Property "CActiveForm.role" is not defined.' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWidgetFactory.php(161): CComponent->__set('role', 'form')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(146): CWidgetFactory->createWidget(Object(SiteController), 'CActiveForm', Array)
#2 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(197): CBaseController->createWidget('CActiveForm', Array)
#3 C:\xampp\htdocs\venturepact\themes\admin\views\site\login.php(18): CBaseController->beginWidget('CActiveForm', Array)
#4 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#5 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(782): CController->renderPartial('login', Array, true)
#8 C:\xampp\htdocs\venturepact\protected\controllers\SiteController.php(153): CController->render('login', Array)
#9 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#10 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(282): CController->run('login')
#19 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#20 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#21 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/venturepact/index.php?r=site/login
---
2015/02/14 11:37:57 [error] [exception.CException] exception 'CException' with message 'Property "CActiveForm.role" is not defined.' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWidgetFactory.php(161): CComponent->__set('role', 'form')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(146): CWidgetFactory->createWidget(Object(SiteController), 'CActiveForm', Array)
#2 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(197): CBaseController->createWidget('CActiveForm', Array)
#3 C:\xampp\htdocs\venturepact\themes\admin\views\site\login.php(18): CBaseController->beginWidget('CActiveForm', Array)
#4 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#5 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(782): CController->renderPartial('login', Array, true)
#8 C:\xampp\htdocs\venturepact\protected\controllers\SiteController.php(153): CController->render('login', Array)
#9 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#10 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(282): CController->run('login')
#19 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#20 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#21 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/venturepact/index.php?r=site/login
---
2015/02/14 11:38:13 [error] [exception.CException] exception 'CException' with message 'Property "CActiveForm.role" is not defined.' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWidgetFactory.php(161): CComponent->__set('role', 'form')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(146): CWidgetFactory->createWidget(Object(SiteController), 'CActiveForm', Array)
#2 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(197): CBaseController->createWidget('CActiveForm', Array)
#3 C:\xampp\htdocs\venturepact\themes\admin\views\site\login.php(18): CBaseController->beginWidget('CActiveForm', Array)
#4 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#5 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(782): CController->renderPartial('login', Array, true)
#8 C:\xampp\htdocs\venturepact\protected\controllers\SiteController.php(153): CController->render('login', Array)
#9 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#10 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(282): CController->run('login')
#19 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#20 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#21 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/venturepact/index.php?r=site/login
---
2015/02/14 11:38:35 [error] [exception.CException] exception 'CException' with message 'Property "CActiveForm.role" is not defined.' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWidgetFactory.php(161): CComponent->__set('role', 'form')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(146): CWidgetFactory->createWidget(Object(SiteController), 'CActiveForm', Array)
#2 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(197): CBaseController->createWidget('CActiveForm', Array)
#3 C:\xampp\htdocs\venturepact\themes\admin\views\site\login.php(18): CBaseController->beginWidget('CActiveForm', Array)
#4 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#5 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(782): CController->renderPartial('login', Array, true)
#8 C:\xampp\htdocs\venturepact\protected\controllers\SiteController.php(153): CController->render('login', Array)
#9 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#10 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(282): CController->run('login')
#19 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#20 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#21 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/venturepact/index.php?r=site/login
---
2015/02/17 05:50:32 [error] [php] include(datatable table table-striped table-bordered table-hover.php): failed to open stream: No such file or directory (C:\xampp\htdocs\Yii\Yii1.1\test\framework\YiiBase.php:432)
Stack trace:
#0 unknown(0): spl_autoload_call()
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\YiiBase.php(303): class_exists()
#2 C:\xampp\htdocs\Yii\Yii1.1\test\framework\YiiBase.php(196): import()
#3 C:\xampp\htdocs\Yii\Yii1.1\test\framework\zii\widgets\grid\CGridView.php(388): createComponent()
#4 C:\xampp\htdocs\Yii\Yii1.1\test\framework\zii\widgets\grid\CGridView.php(359): CGridView->initColumns()
#5 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(147): CGridView->init()
#6 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(180): UsersController->createWidget()
#7 C:\xampp\htdocs\venturepact\protected\modules\admin\views\users\admin.php(61): UsersController->widget()
#8 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(126): require()
#9 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(95): UsersController->renderInternal()
#10 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(869): UsersController->renderFile()
#11 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(782): UsersController->renderPartial()
#12 C:\xampp\htdocs\venturepact\protected\modules\admin\controllers\UsersController.php(135): UsersController->render()
#13 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\actions\CInlineAction.php(49): UsersController->actionAdmin()
#14 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(308): CInlineAction->runWithParams()
#15 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(133): UsersController->runAction()
#16 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilter.php(40): CFilterChain->run()
#17 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(1145): CAccessControlFilter->filter()
#18 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CInlineFilter.php(58): UsersController->filterAccessControl()
#19 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#20 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(291): CFilterChain->run()
#21 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(265): UsersController->runActionWithFilters()
#22 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(282): UsersController->run()
#23 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController()
#24 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#25 C:\xampp\htdocs\venturepact\index.php(13): CWebApplication->run()
REQUEST_URI=/venturepact/index.php?r=admin/users/admin
in C:\xampp\htdocs\venturepact\protected\modules\admin\views\users\admin.php (61)
in C:\xampp\htdocs\venturepact\protected\modules\admin\controllers\UsersController.php (135)
in C:\xampp\htdocs\venturepact\index.php (13)
2015/02/17 05:51:33 [error] [exception.CException] exception 'CException' with message 'Property "CGridView.class" is not defined.' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWidgetFactory.php(161): CComponent->__set('class', 'datatable table...')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(146): CWidgetFactory->createWidget(Object(UsersController), 'zii.widgets.gri...', Array)
#2 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(180): CBaseController->createWidget('zii.widgets.gri...', Array)
#3 C:\xampp\htdocs\venturepact\protected\modules\admin\views\users\admin.php(61): CBaseController->widget('zii.widgets.gri...', Array)
#4 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#5 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(782): CController->renderPartial('admin', Array, true)
#8 C:\xampp\htdocs\venturepact\protected\modules\admin\controllers\UsersController.php(135): CController->render('admin', Array)
#9 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\actions\CInlineAction.php(49): UsersController->actionAdmin()
#10 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(282): CController->run('admin')
#19 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('admin/users/adm...')
#20 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#21 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/venturepact/index.php?r=admin/users/admin
---
2015/02/17 05:54:53 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(282): CController->run('view')
#9 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('admin/suppliers...')
#10 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/venturepact/index.php?r=admin/suppliers/view
---
2015/02/17 05:55:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin/clients".' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('admin/clients')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/venturepact/index.php?r=admin/clients
---
2015/02/17 05:55:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin/usrs".' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('admin/usrs')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/venturepact/index.php?r=admin/usrs
---
2015/02/17 06:03:29 [error] [exception.CException] exception 'CException' with message 'Property "CGridView.itemCssClass" is not defined.' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWidgetFactory.php(161): CComponent->__set('itemCssClass', 'datatable table...')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(146): CWidgetFactory->createWidget(Object(UsersController), 'zii.widgets.gri...', Array)
#2 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(180): CBaseController->createWidget('zii.widgets.gri...', Array)
#3 C:\xampp\htdocs\venturepact\protected\modules\admin\views\users\admin.php(62): CBaseController->widget('zii.widgets.gri...', Array)
#4 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#5 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(782): CController->renderPartial('admin', Array, true)
#8 C:\xampp\htdocs\venturepact\protected\modules\admin\controllers\UsersController.php(135): CController->render('admin', Array)
#9 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\actions\CInlineAction.php(49): UsersController->actionAdmin()
#10 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(282): CController->run('admin')
#19 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('admin/users/adm...')
#20 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#21 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/venturepact/index.php?r=admin/users/admin
---
2015/02/17 06:27:10 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\venturepact\protected\modules\admin\views\users\admin.php:51)
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(869): UsersController->renderFile()
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(782): UsersController->renderPartial()
#2 C:\xampp\htdocs\venturepact\protected\modules\admin\controllers\UsersController.php(135): UsersController->render()
#3 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\actions\CInlineAction.php(49): UsersController->actionAdmin()
#4 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(133): UsersController->runAction()
#6 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CInlineFilter.php(58): UsersController->filterAccessControl()
#9 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(265): UsersController->runActionWithFilters()
#12 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(282): UsersController->run()
#13 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\venturepact\index.php(13): CWebApplication->run()
REQUEST_URI=/venturepact/index.php?r=admin/users/admin
in C:\xampp\htdocs\venturepact\protected\modules\admin\views\users\admin.php (51)
in C:\xampp\htdocs\venturepact\protected\modules\admin\controllers\UsersController.php (135)
in C:\xampp\htdocs\venturepact\index.php (13)
2015/02/18 08:54:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "javascript:;".' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(270): CController->missingAction('javascript:;')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(282): CController->run('javascript:;')
#2 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('admin/users/jav...')
#3 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/venturepact/index.php?r=admin/users/javascript:;
HTTP_REFERER=http://localhost/venturepact/index.php?r=admin/users/admin
---
2015/02/18 18:19:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "clientProjects/admin".' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('clientProjects/...')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/venturepact/index.php?r=clientProjects/admin
HTTP_REFERER=http://localhost/venturepact/
---
2015/02/18 18:19:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "clientProjects/admin".' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('clientProjects/...')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/venturepact/index.php?r=clientProjects/admin
HTTP_REFERER=http://localhost/venturepact/
---
2015/02/18 18:19:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "clientPortfolio/admin".' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('clientPortfolio...')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/venturepact/index.php?r=clientPortfolio/admin
HTTP_REFERER=http://localhost/venturepact/index.php?r=clientProjects/admin
---
2015/02/18 18:19:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "clientProjects/admin".' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('clientProjects/...')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/venturepact/index.php?r=clientProjects/admin
HTTP_REFERER=http://localhost/venturepact/index.php?r=clientPortfolio/admin
---
2015/02/18 18:19:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "suppliers/admin".' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('suppliers/admin')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/venturepact/index.php?r=suppliers/admin
HTTP_REFERER=http://localhost/venturepact/index.php?r=clientProjects/admin
---
2015/02/18 18:22:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "clientProjects/admin".' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('clientProjects/...')
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\venturepact\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/venturepact/index.php?r=clientProjects/admin
HTTP_REFERER=http://localhost/venturepact/
---
2015/02/19 10:15:02 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(282): CController->run('view')
#9 C:\xampp\htdocs\Yii\Yii1.1\test\framework\web\CWebApplication.php(141): CWebApplication->runController('admin/clientPro...')
#10 C:\xampp\htdocs\Yii\Yii1.1\test\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 C:\xampp\htdocs\cloudAdmin\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/cloudAdmin/index.php?r=admin/clientProfiles/view
---
